.github/workflows/build-system.yml

```diff
@@ -22,6 +22,8 @@ jobs:
           container: novelrt/novelrt-build:latest
           profile: linux-clang10-amd64
           name: Ubuntu - Clang 10
+          c: CC=clang-10
+          cxx: CXX=clang++-10
```

Comment:
Because we're no longer using Conan to define the compiler, we're explicitly defining it for Ubuntu as it is not the stock default one for GH's runners. We can optionally exclude it (as we do for macOS and Windows) to let GH compile with what is default.